src/renderers/noop/ReactNoop.js

```diff
@@ -136,10 +136,22 @@ var NoopRenderer = ReactFiberReconciler({
   },
 
   scheduleAnimationCallback(callback) {
+    if (scheduledAnimationCallback) {
+      throw new Error(
+        'Scheduling an animation callback twice is excessive. ' +
+        'Instead, keep track of whether the callback has already been scheduled.'
+      );
```

Comment:
I see, if it's only the noop renderer then it makes sense. Just wouldn't want to throw in a public renderer for a case that's unlikely to produce an error.